core/editor/src/input/input_mapper.rs

```diff
@@ -235,6 +235,10 @@ impl Default for Mapping {
 			entry! {action=DocumentMessage::NudgeSelectedLayers(NUDGE_AMOUNT, -NUDGE_AMOUNT), key_down=KeyArrowRight, modifiers=[KeyArrowUp]},
 			entry! {action=DocumentMessage::NudgeSelectedLayers(NUDGE_AMOUNT, NUDGE_AMOUNT), key_down=KeyArrowRight, modifiers=[KeyArrowDown]},
 			entry! {action=DocumentMessage::NudgeSelectedLayers(NUDGE_AMOUNT, 0.), key_down=KeyArrowRight},
+			entry! {action=DocumentMessage::MoveSelectedLayer(-1), key_down=KeyLeftBracket, modifiers=[KeyControl]},
+			entry! {action=DocumentMessage::MoveSelectedLayer(1), key_down=KeyRightBracket, modifiers=[KeyControl]},
+			entry! {action=DocumentMessage::MoveSelectedLayer(-999999), key_down=KeyLeftBracket, modifiers=[KeyControl, KeyShift]},
+			entry! {action=DocumentMessage::MoveSelectedLayer(999999), key_down=KeyRightBracket, modifiers=[KeyControl, KeyShift]},
```

Comment:
Please use `i32::MAX` and `i32::MIN` instead.

Reply:
That should cause the addition in `DocumentMessageHandler.process_action` to overflow... but I just tested it and it doesn't somehow? I don't like it when code works but I don't know why :thinking:

This is the behaviour I expected: (panic with 'attempt to add with overflow') https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=91a719d050d320467beaf56c7a2dcda5

We can also special case these two values if that's preferable to using arbitrarily large ints here.

Reply:
You should either add a special case or cast through i64. Casting is probably better because the user could still give weird input that overflows otherwise